hadoop-hdds/docs/content/concept/Containers.zh.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Container
+weight: 5
+menu: 
+  main:
+     parent: 架构
```

Comment:
The rename of the parent menu I think can be handled in the new issue because it also contains other items.
Inconsistent changes will cause some menus to disappear.